src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -308,6 +314,17 @@
  (fn [{:keys [db]} [chain-ids]]
    {:db (assoc-in db [:wallet :ui :send :disabled-from-chain-ids] chain-ids)}))
 
+(rf/reg-event-fx :wallet/lock-from-amount
+ (fn [{:keys [db]} [chain-id amount]]
+   {:db (assoc-in db [:wallet :ui :send :from-locked-amounts chain-id] amount)}))
+
+(rf/reg-event-fx :wallet/unlock-from-amount
+ (fn [{:keys [db]} [chain-id]]
+   (let [new-locked-amounts (-> db
+                                (get-in [:wallet :ui :send :from-locked-amounts])
+                                (dissoc chain-id))]
+     {:db (assoc-in db [:wallet :ui :send :from-locked-amounts] new-locked-amounts)})))
+
```

Comment:
We actually want to update the entry, so `update-in` could make this code easier to grasp:

```clojure
{:db (update-in db [:wallet :ui :send :from-locked-amounts] dissoc chain-id)}
```

wdyt?

Reply:
Yes, this is much better, thank you!